client/routes/domain/workflow-archival/helpers/map-archived-workflow-response.spec.js

```diff
@@ -0,0 +1,40 @@
+import mapArchivedWorkflowResponse from './map-archived-workflow-response';
+
+describe('mapArchivedWorkflowResponse', () => {
+  describe('When passed nextPageToken = "123"', () => {
```

Comment:
Do you need extra level of describe? Maybe reformat to:
- it('returns nextPageToken when passed one', ...
- it('returns flattened results array', ...

Reply:
can do